### PR TITLE
remove request_locked_pdf_password feature flag

### PR DESCRIPTION
**Pull request by jaredcunha**
## Description
Removes the old `request_locked_pdf_password` feature flag

## Original issue(s)
department-of-veterans-affairs/va.gov-team#38842

## Testing done
Tests still pass

## Acceptance criteria
- [x] Remove feature flag `request_locked_pdf_password feature` code in 526
- [x] Update tests

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: src/platform/forms-system/src/js/fields/FileField.jsx [hunk 1]
```diff
@@ -324,8 +321,7 @@ class FileField extends React.Component {
               );
               const attachmentNameErrors = get([index, 'name'], errorSchema);
               // feature flag
-              const showPasswordContent =
-                requestLockedPdfPassword && file.isEncrypted;
+              const showPasswordContent = file.isEncrypted;
```
Comment (bkjohnson) [inline src/platform/forms-system/src/js/fields/FileField.jsx:324]: The `// feature flag` comment above can go now. Additionally, with the feature flag gone `showPasswordSuccess` seems a little hard to follow now:

https://github.com/department-of-veterans-affairs/vets-website/blob/092178c895ca9bfa0acbf084c45b698f2e04373d/src/platform/forms-system/src/js/fields/FileField.jsx#L329-L334

If `showPasswordContent` is just `file.isEncrypted`, then we basically have:

```js
const showPasswordInput =
    file.isEncrypted && !file.confirmationCode;
const showPasswordSuccess =
    file.isEncrypted &&
    !(file.isEncrypted && !file.confirmationCode) &&
    file.confirmationCode;
```

That middle boolean expression might not be necessary anymore since both `isEncrypted` and `confirmationCode` have to be truthy for the whole expression to be true. I think we could simplify to:

```js
const showPasswordInput =
    file.isEncrypted && !file.confirmationCode;
const showPasswordSuccess =
    file.isEncrypted && file.confirmationCode;
```

Comment (jaredcunha) [reply src/platform/forms-system/src/js/fields/FileField.jsx]: Good catch! I cleaned up the file a little bit, which I _think_ should address your comment. If not, let me know.

File: src/applications/static-pages/contact-chatbot-cta/components/App/index.unit.spec.js [hunk 2]
```diff
@@ -6,12 +6,6 @@ import { shallow } from 'enzyme';
 import { App } from '.';
 
 describe('Contact Chatbot CTA <App>', () => {
-  it('does not render when feature toggle is falsey', () => {
-    const wrapper = shallow(<App show={false} />);
-    expect(wrapper.type()).to.equal(null);
-    wrapper.unmount();
-  });
-
```
Comment (jaredcunha) [inline src/applications/static-pages/contact-chatbot-cta/components/App/index.unit.spec.js:14]: This test was failing after removing the feature flag, but I'm sort of unsure if I should have deleted it.

File: src/platform/forms-system/src/js/fields/FileField.jsx (hunk 1 above)
Comment (bkjohnson) [reply src/platform/forms-system/src/js/fields/FileField.jsx]: Looks good to me :+1:

**Review by bkjohnson — APPROVED** [review]

**Review by theodur — APPROVED** [review]

**Review by batemapf — APPROVED** [review]

**Comment by it-harrison** [comment]
if you sync with `main` the unit tests should pass

**Review by k80bowman — APPROVED** [review]

**Review by it-harrison — APPROVED** [review]

**Comment by Mottie** [comment]
@department-of-veterans-affairs/vsa-debt-frontend @department-of-veterans-affairs/afs-code-reviewers Please review this PR

**Comment by Mottie** [comment]
@AdamFreemer @dsrr47 @PJart @af-manish @dan-brooking or @abuchanan83 - would one of you please review this PR please.

**Review by OptionSelect — APPROVED** [review]
Looks great to me! 🚀
